cmd/bundlectl/integration-tests/version.sh

```diff
@@ -0,0 +1,3 @@
+
+bundlectl="./$1"
```

Comment:
+ #!/bin/bash
+ copyright header 

set -o errexit
set -o nounset
set -o pipefail

Reply:
this is specifically what I meant about following the bash style guide.